packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -324,6 +332,46 @@ export function getPublishResolver<TSource, TContext, TArgs>({
   }
 }
 
+/**
+ * Returns values for link arguments, also covers the cases for
+ * if the link parameter contains constants that are appended to the link parameter
+ *
+ * e.g. instead of:
+ * $response.body#/employerId
+ *
+ * it could be:
+ * abc_{$response.body#/employerId}
+ */
+function inferLinkArguments<TSource, TContext, TArgs>({
+  paramName,
+  value,
+  resolveData,
+  source,
+  args
+}: inferLinkArgumentsParam<TSource, TContext, TArgs>)  {
+  if (Object.prototype.toString.call(value) === '[object Object]') {
```

Comment:
I don't really like this pattern. It seems a bit lengthy and we don't use this pattern anywhere else in our code base. I believe that `typeof value === 'object'` should be sufficient for this use case. Feel free to correct me if I am wrong.

Reply:
using the prototype method might be overkill, but it does guarantee we are dealing with a real object but I think `typeof` should suffice in this case.